ydb/_topic_writer/topic_writer_asyncio.py

```diff
@@ -554,7 +557,8 @@ async def flush(self):
             return
 
         # wait last message
-        await asyncio.wait((self._messages_future[-1],))
+        await asyncio.wait(self._messages_future)
+        self._messages_future = deque()
```

Comment:
re-create _messages_future here is bug: we wait features on start waiting, but while we wait - somebody can add new messages from other coroutine.

And don't need in happy way - it will cleaned early within _handle_receive_ack be popleft future for every message.

Reply:
right